src/main/java/com/google/api/codegen/go/GoDiscoveryContext.java

```diff
@@ -110,6 +111,11 @@ private String typeName(Type type, Field field) {
   }
 
   @Override
+  public boolean isResponseEmpty(Method method) {
```

Comment:
Why do we need to override this? Can't we just let DiscoveryContext.java handle this?